scripts/master_node/download_images.sh

```diff
@@ -24,36 +24,37 @@ else
 fi
 
 ARCH=$(uname -m)
-VERSION="latest"
-PIP=pip.conf.bak
+VERSION="${VERSION:-latest}"
 
 echo_b "Downloading the docker images for Cello services: VERSION=${VERSION} ARCH=${ARCH}"
 
+# TODO: will be removed after we have the user dashboard image
+echo_b "Check node:9.2 image."
+[ -z "$(docker images -q node:9.2 2> /dev/null)" ] && { echo "pulling node:9.2"; docker pull node:9.2; }
+
 # docker image
 
-for IMG in dashboard api-engine; do
+for IMG in dashboard nginx api-engine; do
 	HLC_IMG=hyperledger/cello-${IMG}
-	PATH=build_image/docker/common/${IMG}/Dockerfile.in
-	if [ -z "$(docker images -q ${HLC_IMG}:latest 2> /dev/null)" ]; then  # not exist
-	echo_b "Build ${HLC_IMG} locally"
-	docker build -t ${HLC_IMG}:latest -f ${PATH} ./ --build-arg pip=${PIP} --platform linux/${ARCH}
-	#hyperledger/cello-<service>:latest -f build_image/docker/common/api-engine/Dockerfile.in ./ --platform linux/$(ARCH); \
-	# docker tag ${HLC_IMG}:${ARCH}-${VERSION} ${HLC_IMG}  # match the docker-compose file
-	else
-		echo_g "${HLC_IMG} already exist locally"
-	fi
+	#if [ -z "$(docker images -q ${HLC_IMG}:${ARCH}-${VERSION} 2> /dev/null)" ]; then  # not exist
+	echo_b "Pulling ${HLC_IMG}:${ARCH}-${VERSION} from dockerhub"
+	docker pull ${HLC_IMG}:${ARCH}-${VERSION}
+	docker tag ${HLC_IMG}:${ARCH}-${VERSION} ${HLC_IMG}  # match the docker-compose file
+	#else
+	#	echo_g "${HLC_IMG} already exist locally"
+	#fi
 done
 
 # We now use official images instead of customized one
-# docker pull mongo:3.4.10
+docker pull mongo:3.4.10
```

Comment:
what's the purpose of changing download_images.sh? Why need to pull these images, which are not used.